### ISSUE TITLE
Day 9 Exercise 1 Question 2 doesn't make sense

### ISSUE THREAD

**Issue by Ngoakor12** [Asabeneh/30-Days-Of-JavaScript#170]
Here's the current question:
2. Define a call function before you them in forEach, map, filter or reduce.

Is there supposed to be  "use" between "you" and "them"?

If it's not a grammatical error can you rephrase the question?

PS. this is my first issue ever, so be nice.

**Comment by kenliten** [comment]
I think it actually refers to callback in call and I guess, the final sentence make sense like this:

2. Define a callback function before you use it in forEach, map, filter or reduce.

However, now I'm going to submit that change,
Thanks, @Ngoakor12
